ipatests/test_xmlrpc/test_add_remove_cert_cmd.py

```diff
@@ -352,3 +352,94 @@ def delete_entity(cls):
             api.Command.host_del(TestCertManipCmdHost.entity_pkey)
         except errors.NotFound:
             pass
+
+
+@pytest.mark.tier1
+class TestCertManipIdOverride(XMLRPC_test):
+    idview = u'testview'
+    testuser = u'testuser'
+    entity_subject = testuser
+    entity_principal = testuser
+
+    cert_add_cmd = api.Command.idoverrideuser_add_cert
+    cert_del_cmd = api.Command.idoverrideuser_remove_cert
+
+    def del_cert_from_idoverride(self, username, view_name, cert):
+        result = self.cert_del_cmd(view_name,
+                                   username,
+                                   "--certificate=%s" % cert)
+        return dict(
+            usercertificate=result['result'].get('usercertificate', []),
+            value=result.get('value'),
+            summary=result.get('summary')
+        )
+
+    def add_cert_to_idoverride(self, username, view_name, cert):
+        result = self.cert_add_cmd(view_name,
+                                   username,
+                                   usercertificate=cert)
+        return dict(
+            usercertificate=result['result'].get('usercertificate', []),
+            value=result.get('value'),
+            summary=result.get('summary')
+        )
+
+    @classmethod
+    def teardown_class(cls):
+        api.Command.user_del(cls.testuser)
+        api.Command.idview_del(cls.idview)
+        super(TestCertManipIdOverride, cls).teardown_class()
+
+    @classmethod
+    def setup_class(cls):
```

Comment:
The same reasoning as in teardown. Fixtures are a better place for resources needed by the test (and are reusable)